dask_planner/src/sql.rs

```diff
@@ -381,9 +381,7 @@ impl DaskSQLContext {
             DaskStatement::CreateModel(create_model) => Ok(LogicalPlan::Extension(Extension {
                 node: Arc::new(CreateModelPlanNode {
                     model_name: create_model.name,
-                    input: self._logical_relational_algebra(DaskStatement::Statement(Box::new(
-                        create_model.select,
-                    )))?,
+                    input: self._logical_relational_algebra(create_model.select)?,
```

Comment:
Ideally, we should be doing a match on `create_model.select` to make sure it's a relevant variant - the main ones I had in mind were `Statements` and `PredictModels`, but think it could realistically be any variant that outputs a table?